Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings

```diff
@@ -986,7 +968,7 @@
 "login.loginInfoDetailsWebsiteFieldTitle" = "Web sitesi";
 
 /* The message displayed in alert when a user needs to set a passcode */
-"login.loginInfoSetPasscodeAlertDescription" = "Şifreleri görmek için, öncelikle cihazınızda bir parola belirlemeniz gerekiyor.";
+"login.loginInfoSetPasscodeAlertDescription" = "Senkronizasyon zinciri ayarlamak veya ayarları görmek için, öncelikle cihazınızda bir parola belirlemeniz gerekiyor.";
```

Comment:
Senkronizasyon zinciri`ni` ayarlamak veya ayarları görmek için, öncelikle cihazınızda bir parola belirlemeniz gerekiyor.